compiler/packages/babel-plugin-react-compiler/src/Entrypoint/Pipeline.ts

```diff
@@ -314,6 +316,15 @@ function runWithEnvironment(
     value: hir,
   });
 
+  if (env.config.enableReactiveGraph) {
+    const reactiveGraph = buildReactiveGraph(hir);
+    log({
+      kind: 'debug',
+      name: 'BuildReactiveGraph',
+      value: printReactiveGraph(reactiveGraph),
+    });
+  }
```

Comment:
note that i'm experimenting with running this before constructing scope terminals, since that's where it would have to go in order to reorder to avoid unnecessary scope interleaving